LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-invalid.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Level 3: parsing align-tracks with invalid values</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-3/#tracks-alignment">
+<meta name="assert" content="align-tracks supports only the grammar 'normal | <baseline-position> | <content-distribution> | <overflow-position>? <content-position>'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_invalid_value("align-tracks", "auto");
```

Comment:
Would be nice to see more list-related tests, especially a testcase with a valid value everywhere, but the first, middle, last one being invalid.